apps/compartmentConverter.cpp

```diff
@@ -61,6 +63,19 @@ template< class T > void requireEqualCollections( const T& a, const T& b )
     REQUIRE_EQUAL( j, b.end( ));
 }
 
+/** @return true if the cell occupies a continuous region in the report frame */
+bool _isContinuous( const brion::CompartmentReport& report,
+                    const size_t gidIndex )
+{
+    const auto& offsets = report.getOffsets()[gidIndex];
+    const auto& counts = report.getCompartmentCounts()[gidIndex];
+    for( size_t i = 0; i < offsets.size() - 1; ++i )
+        if( offsets[i] + counts[i] != offsets[i+1] )
+            return false;
```

Comment:
Rename to _isContiguous or _isCompact

This is not completely accurate. A cell could be in a single contiguous region, but since the sections are not guaranteed to be enumerated in order inside the region, this function would return a false negative. It's more correct to do the following: let's call  [Si_a, Si_b] the offset range where a section Si maps (Si_a is the base offset and Si_b is Si_a + num compartments of that section), what we need to verify is that for any section Sx, [Sx_a, Si_b] is in [min(Si_a), max(Si_b)] and that max(Si_b) - min(Si_b) = num compartments for this cell. The second part is the key to ensure that all sections are compact.